### PR TITLE
Release/0.12.1

### PR DESCRIPTION
**Pull request by ProofOfKeags**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -7,7 +7,7 @@ ADD . /root
 
 WORKDIR /root/lnd
 
-RUN make install tags="signrpc walletrpc chainrpc invoicesrpc"
+RUN make -j24 install tags="autopilotrpc signrpc walletrpc chainrpc invoicesrpc routerrpc watchtowerrpc"
```
Comment (ProofOfKeags) [inline Dockerfile:10]: add all the rpc's explicitly

File: configurator/src/main.rs [hunk 2]
```diff
@@ -389,14 +389,19 @@ fn main() -> Result<(), anyhow::Error> {
...
+                let bs = std::fs::read(channel_backup_path)?;
+                std::fs::remove_dir_all("/root/.lnd/data/graph")?;
+                let encoded = base64::encode(bs);
+                Ok::<Option<Value>, std::io::Error>(Some(serde_json::json!({
+                    "multi_chan_backup": encoded
+                })))
+            } else {
+                println!("No channel restoration required. No channel backup exists.");
+                Ok(None)
+            }
```
Comment (ProofOfKeags) [inline configurator/src/main.rs:404]: If the channel backup doesn't exist, there is no risk of fund loss, so we can safely take no action.

File: manifest.yaml [hunk 3]
```diff
@@ -31,7 +35,7 @@ assets: []
 hidden-service-version: v3
 dependencies:
   btc-rpc-proxy:
-    version: "^0.2.4"
+    version: ">=0.2.4 <0.4.0"
```
Comment (ProofOfKeags) [inline manifest.yaml:38]: Relax the upper bound on the Proxy dependency

**Review by chrisguida — APPROVED** [review]
configurator crashes on old line  392-394 if channel backup doesn't exist. channel backup will not exist if the user has deposited funds in their on-chain wallet but hasn't opened any channels.

**Pull request merged**